packages/nx-playwright/src/executors/playwright-executor/executor.ts

```diff
@@ -14,6 +14,7 @@ function getFlags(options: PlaywrightExecutorSchema) {
   const grepOption = options.grep !== undefined ? `--grep=${options.grep}` : '';
   const grepInvertOption =
     options.grepInvert !== undefined ? `--grep-invert=${options.grepInvert}` : '';
+  const debugOption = options.debug !== undefined ? '--debug' : '';
```

Comment:
Why not just `const debugOption = options.debug ? '--debug' : '';` (providing debug is available as boolean here and it's not a string)

I guess you are following the pattern used for other attributes  but I'm not sure why it was done this way to be honest